### ISSUE TITLE
Update Social Media Login config examples

### ISSUE THREAD

**Issue by Rwalls1** [OpenLiberty/docs#4439]
Doc: https://draft-openlibertyio.mybluemix.net/docs/latest/reference/feature/socialLogin-1.0.html
Comments from Karen related to issue #2075:
- I believe the google ID for the "Request log in..." section is represented by "your_app_id" for clientId. Would you work in "your_app_id" and/or clientId into your description to give the user more context?
- "with the available providers" seems like it is missing some info/dangling. Maybe change "with the available providers" to "with the available providers included on the form.
- Globally change "app" to "application".
- Change "...so that users can have the option of logging in with ..." to "...so that users can log in with ..." to reduce wordiness.
- Rewrite to remove "out-of-the-box" from the "Provide other social media logins..." section, and which is flagged by Acrolinx. 
- Rerun Acrolinx and fix items as appropriate.
- For "For more information, see the authFilter configuration element.", fix the broken link. Also, we've found that highlghting in links is confusing. So, remove the highlighting from "authfFilter".
- The same goes for oauth2Login and oidcLogin 
- Change "The socialLogin-1.0 feature can be configured " to "The socialLogin feature can be configured " and remove the highlighting. A new version of the feature can become available. So, we don't want to specify a particular version, if possibl

**Comment by ramkumar-k-9286** [comment]
David. Multiple links in this document are broken. i cant locate the proper links to replace the broken links. Who do i contact for this.

Rest of the updates requested have been done. need to correct the links before proceeding.

@dmuelle

**Comment by dmuelle** [comment]
Hi Ram- 

here are the fixes for the links

**change**
see the LDAP User Registry (feature:ldapRegistry[]) feature.
**to**
see the feature:ldapRegistry[display=LDAP User Registry feature].

**change**
see the link:https://docs/ref/config/#authFilter.html[`authFilter`] configuration element..
**to**
see the config:authFilter[display=authFilter element].

**change**
use the link:https://docs/ref/config/#oauth2Login.html[`oauth2Login`] element
**to**
use the config:oauth2Login[display=oauth2Login element] 

**change**
or the link:https://docs/ref/config/#oidcLogin.html[`oidcLogin`] element
**to**
or the config:oidcLogin[display=oidcLogin element] 


Since these links are all to either feature or config reference pages, they should use macro syntax, as shown in my edit suggestions. This page had not been updated for a while so the link syntax was out of date. For more info about the macro syntax, see the wiki entry for [Technical linking information](https://github.com/OpenLiberty/docs/wiki/Stylistic-guidelines#technical-linking-information). Let me know if you have any questions.

**Comment by dmuelle** [comment]
Hi Ram, I've verified the draft against the changes outlined in this issue. Everything looks good except there is a typo to fix:

For more information, sesee the authFilter element.


Also, I think you should run Acrolinx on the entire topic (except for the code examples) one more time. If you need help or have any questions, let me know- thanks.

**Comment by ramkumar-k-9286** [comment]
@dmuelle 

Draft document link:
https://draft-openlibertyio.mybluemix.net/docs/21.0.0.9/reference/feature/socialLogin-1.0.html


Regards,
Ramkumar

**Comment by dmuelle** [comment]
Hi Ram- the doc looks good! All the items mentioned in this issue have been addressed. However, when I reviewed the doc just now I noticed a problem that no one has mentioned yet, and that Acrolinx doesn't catch. In the following headings in the topic, there is some confusion about "log in" vs "login". 

`Request log in with a social media ID`
`Request log in with a social media ID or with their account for the configured registry`
`Request log in with a social media ID for only a subset of applications, URLs, browsers, or IP addresses`

In each of these headings, "login" is used as a noun, so should be one word instead of two. For example:
`Request login with a social media ID`


 The other option would be to change the heading so that "log in" is a verb. For example:
`Request that users log in with a social media ID`

I think either option can work, the first is more concise, but the second might be more clear. Maybe we can discuss briefly before you update it. The topic is otherwise ready to go I think.
